#!/usr/bin/python
PACKAGE = "ros_adas2019_examples"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("startX", double_t, 0, "search line start in x direction (relative, from left)", 0.58, 0, 1.0)
gen.add("endX", double_t, 0, "search line end in x direction (relative, from left)", 0.78, 0, 1.0)
gen.add("searchY", double_t, 0, "search line y direction (relative, from top)", 0.625, 0, 1.0)
gen.add("debugImageScaling", double_t, 0, "image scaling for low bandwidth remote visualization", 0.2, 0.1, 1.0)

exit(gen.generate(PACKAGE, "ros_adas2019_examples", "SimpleLineFollower"))
